_wiki/basic-tutorials/logger-configuration.md

```diff
@@ -2,39 +2,64 @@
 
-## Fallback Logger Configuration
+## Fallback Logger
+
+Javacord's fallback logger is a simple Log4j logger which always logs `INFO` level and higher. It allows you to enable `DEBUG` and `TRACE` logging manually. As log levels are hierarchical, enabling `TRACE` will also implicitly enable `DEBUG` and disabling `DEBUG` will also implicitly disable `TRACE`.
 
-Javacord's fallback logger is a very simple slf4j implementation which always logs `INFO` level and higher. It allows you to enable `DEBUG` and `TRACE` logging manually:
 ```java
 // Enable debug logging
 FallbackLoggerConfiguration.setDebug(true);
 
 // Enable trace logging
 FallbackLoggerConfiguration.setTrace(true);
 ```
-All messages are printed to the standard output stream (`System.out`) or the standard error output stream (`System.err`) respectively.
 
-## Using SLF4J
+Changing the log level of the fallback logger only has any effect on newly created loggers. Pre-existing loggers will not have their log level changed. So if you want to configure the fallback logger, you should do this as one of the first things in your bot code. If you want to change log levels during runtime, you should use a proper logging framework like Log4j 2 Core that supports this.
+
+All messages of the fallback logger are printed to the standard output stream (`System.out`) and thus usually to your console. If you want to log to a file, database or anything else, you should consider using a proper logging framework which allows you to configure this behavior.
+
+This is how a log line of the fallback logger will look like:
+
+```log
+<time with date            ><level><logger name, usually the logging class              > <message            > <the thread context, here the shard number>
+2018-08-03 20:00:06.080+0200 DEBUG org.javacord.core.util.gateway.DiscordWebSocketAdapter Received HELLO packet {shard=0}
+```
+
+## Using a Proper Logging Framework
 
 ### Adding a Logging Framework
 
-Adding a logging framework of your choice is very straightforward. You can just add it as a dependency and it will be detected by slf4j automatically. This following example adds logback-classic using Maven:
-```xml
-<dependency>
-  <groupId>ch.qos.logback</groupId>
-  <artifactId>logback-classic</artifactId>
-  <version>1.2.3</version>
-</dependency>
+Adding a logging framework of your choice is very straightforward. You can just add it as a dependency and it will be detected by Log4j automatically. The following example adds Log4j 2 using Gradle:
+
+```groovy
+dependencies { runtimeOnly 'org.apache.logging.log4j:log4j-core:2.11.0' }
+```
+
+You can also use and SLF4J compatible logging framework using `log4j-to-slf4j`. The following example adds Logback Classic using Gradle:
```

Comment:
`also use and SLF4J` is a typo, it should be `also use an SLF4J`

Reply:
neither, should be `a SLF4J compatible`

Reply:
maybe. I use it because it's pronounced as `an ess ell eff four jay`

Reply:
@Kaleidox00 just stop commenting, please.
As Walker says, you use `an` if you **speak** a vowel, doesn't matter at all whether there is a vowel written or not.
That's also what it often makes it hard for non-native speakers to get it right if they don't know how to pronounce a word.

Reply:
you dont _speak_ a vowel in a shortcut, except the first letter of the shortcut is a vowel, and even then it would be quite sick

Reply:
Yes you do.
http://blog.apastyle.org/apastyle/2012/04/using-a-or-an-with-acronyms-and-abbreviations.html
**If** you speak the abbreviation with a vowel in the beginning you use `an`.